static/js/video-apis/VimeoAPI.js

```diff
@@ -0,0 +1,21 @@
+import Player from '@vimeo/player';
+
+/**
+ * Interacts with the Vimeo Player API.
+ */
+class VimeoAPI {
+  /**
+   * Adds a new Vimeo iframe player using the given element.
+   *
+   * @param {HTMLElement} el 
+   * @param {Function} onPlay 
```

Comment:
nit: I think we should indicate that the second param is an object like:
```
@param {HTMLElement} el 
@param {Object} options
@param {Function) options.onPlay
```
Thoughts?

Reply:
will call the param param1 instead of options since that's what my vscode spit out